src/iTunesLibrary/Enums.cs

```diff
@@ -301,10 +301,15 @@ public enum ITLibPlaylistProperty {
 		AllItemsPlaylist,
 		[Field ("ITLibPlaylistPropertyDistinguisedKind")]
 		DistinguisedKind,
+		[Introduced (PlatformName.MacOSX, 10, 6, message: "Use 'ITLibPlaylistPropertyPrimary' instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'ITLibPlaylistPropertyPrimary' instead.")]
```

Comment:
This one is also not needed

```suggestion
		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'ITLibPlaylistPropertyPrimary' instead.")]
```